hw3/slavrom/constructorFunc.js

```diff
@@ -0,0 +1,73 @@
+import { SarahMay, JeremieBrown, Elizabeth } from '../people.data.js';
+
+function PersonFunc(person) {
+    this.person = person;
+}
+
+PersonFunc.prototype.getFullName = function() {
+  const fullName = `${this.person.firstName} ${this.person.lastName}`;
+  if (!this.person.firstName) {
```

Comment:
same note as in class version: if we have just firstName would be better to return this first name and  in case when we have just lastName - throw error
oh, and try...catch applicable here as well but it's not necessary to change 😉